src/pcp/summary/pcp-summary.sh

```
@@ -433,9 +443,9 @@ echo " hardware: "`echo $hardware | _fmt`
 echo " timezone: $timezone"
 [ -n "$services" ] && echo " services: $services"
 
-echo "     pmcd: ${version},${numagents}$numclients"
+[ -n "$version" ] && echo "     pmcd: ${version},${numagents}$numclients"
 
-[ -n "$agents" ] && echo "     pmda: $agents"
+[ -n "$version" -a -n "$agents" ] && echo "     pmda: $agents"
```

Shellcheck (0.11.0)

[warning] Prefer [ p ] && [ q ] as [ p -a q ] is not well defined.

(SC2166)